homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,45 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+
+    int num0, num1;
+    if(lhs < 0){
+        lhs *= (-1);
+    }
+    if(rhs < 0) {
+        rhs *= (-1);
+    }
+    if(lhs < rhs) {
+        num0 = lhs;
+        lhs = rhs;
+        rhs = num0;
+    }
+    if(rhs == 0) {
+        return lhs;
+    }
+    if(lhs % rhs == 0) {
+        return rhs;
+    }
```

Comment:
Use std::swap instead create new variable

if (lhs > rhs) {
   std::swap(lhs, rhs)
}

you can also use abs